hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,196 @@
+                        className={styles.cartButton}
+                        type="submit"
+                        onClick={handleSubmit}
+                        disableElevation
+                    >
+                        ADD TO CART
+                    </Button>
+                </form>
+            </>
+        )
+    );
+};
+
+export const EnhancedCartForm = ({ handleCart, requestFailure, availableQuantity }) => {
+    const onSubmit = (formikValues) => {
+        handleCart(formikValues.quantity);
+    };
+
+    return (
+        handleCart && (
+            <Formik
+                initialValues={{ quantity: "1" }}
+                onSubmit={onSubmit}
+                validateOnBlur={false}
+                validationOnChange={false}
+                validationSchema={false}
+            >
+                {(formikProps) => (
+                    <CartForm
+                        availableQuantity={availableQuantity}
+                        handleSubmit={formikProps.handleSubmit}
+                        handleChange={formikProps.handleChange}
+                        requestFailure={requestFailure}
+                        values={formikProps.values}
+                    />
+                )}
+            </Formik>
+        )
+    );
+};
+
+const BodySheet = ({ manufacturer, model_num, datasheet, notes, constraints }) => {
+    return (
+        <div className={styles.bodysheet}>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2">Manufacturer</Typography>
+                <Typography>{manufacturer}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2">Model Number</Typography>
+                <Typography>{model_num}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2">Datasheet</Typography>
+                <Link
+                    href={datasheet}
+                    rel="noopener"
+                    color="inherit"
+                    underline="none"
+                    target="_blank"
+                    className={styles.bodyinfoDataSheet}
+                >
+                    <LaunchIcon></LaunchIcon>
+                    <Typography>Link</Typography>
+                </Link>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2">Notes</Typography>
+                {notes.map((note, i) => (
+                    <Typography key={i}>{note}</Typography>
+                ))}
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2">Constraints</Typography>
+                {constraints.map((constraint, i) => (
+                    <Typography key={i}>{constraint}</Typography>
+                ))}
+            </div>
+        </div>
+    );
+};
+
+const HeaderSheet = ({ type, name, total, available, categories, img }) => {
```

Comment:
Is this something that's standard across all the drawers we'll have (cc @lisa-sa-li)? If so, it could be standardized and moved into `SideSheetRight.js` as well

Reply:
Seems I was a little bit confused about what this is. Could you give it a name other than `HeaderSheet`, since it is more than just the header